app/Http/Controllers/Admin/BlackoutController.php

```diff
@@ -143,7 +144,7 @@ public function createBlackoutForEveryRoom(Request $request)
   {
       $request->validateWithBag('createBlackoutAll', [
           'start_date' => ['required', 'date'],
-          'end_date' => ['required', 'date', "after:start"],
+          'end_date' => ['required', 'date', self::AFTER_START],
```

Comment:
think this should've been after start_time

Reply:
oh rip @EvanDime the auto merge kicked in

Reply:
dude...you don't approve if you don't think it can merge xD
Either way is something we can fix later. this was just to get the smell away.  Also we can now just update teh constant and all the instances will work too